CMakeLists.txt

```diff
@@ -1,161 +1,74 @@
 # CMAKE project for openrct2
-cmake_minimum_required(VERSION 3.1)
+cmake_minimum_required(VERSION 3.9)
 if (CMAKE_BINARY_DIR STREQUAL CMAKE_SOURCE_DIR)
     message(FATAL_ERROR "Building in-source is not supported! Create a build dir and remove ${CMAKE_SOURCE_DIR}/CMakeCache.txt")
 endif()
 
-project(openrct2)
+project(openrct2 VERSION 0.1.5 LANGUAGES C CXX)
 
-include(FindPkgConfig)
-include(CheckCCompilerFlag)
-include(CheckCXXCompilerFlag)
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/cmake")
+
+# Put all runtime binaries in <build>/bin
+set(CMAKE_RUNTIME_OUTPUT_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/bin")
+# Copy data into the bin dir. This permits running easily out of the build directory.
+file(COPY data DESTINATION bin)
```

Comment:
When is it getting copied? During configuration (`cmake ../`) or installation (`make install`)? Will it fail if I already have it symlinked?